src/wagtail_live/webapp/receiver.py

```diff
@@ -41,7 +43,33 @@ def get_message_text(self, message):
     def get_message_files(self, message):
         """See base class."""
 
-        return message["files"] if "files" in message else []
+        return message["images"]
+
+    def get_image_title(self, image):
+        """See base class."""
+
+        return image["image"]["name"].replace("-", " ")
```

Comment:
Is it maybe nice to also remove the extension from the name here? (I'm assuming it's there because of the get_image_mimetype method)

Reply:
Alright!